src/components/StageEditor/sections/SociogramPrompts/SociogramPrompt.js

```diff
@@ -0,0 +1,287 @@
+      backgroundType: CONCENTRIC_CIRCLES,
+    };
+  }
+
+  handleChooseBackgroundType = (option) => {
+    this.setState({ backgroundType: option });
+  }
+
+  handleHighlightOrCreateEdge = (type) => {
+    switch (type) {
+      case HIGHLIGHT:
+        return this.props.clearField(`${this.props.fieldId}.edges.create`);
+      case CREATE_EDGE:
+        return this.props.clearField(`${this.props.fieldId}.highlight.allowHighlighting`);
+      default:
+        return null;
+    }
+  }
+
+  clearEmptyField = (event, value, previousValue, name) => {
+    if (isEmpty(value)) {
+      this.props.clearField(name);
+      event.preventDefault();
+    }
+  }
+
+  render() {
+    const { backgroundType } = this.state;
+    const {
+      fieldId,
+      nodeTypes,
+      edgeTypes,
+      variablesForNodeType,
+      layoutsForNodeType,
+      highlightableForNodeType,
+      ...rest
+    } = this.props;
+
+    return (
+      <ExpandableItem
+        className="stage-editor-section-sociogram-prompt"
+        open={this.props.isDirty}
+        preview={(
+          <FormSection name={fieldId}>
+            <div className="stage-editor-section-sociogram-prompt__preview">
+              <div className="stage-editor-section-sociogram-prompt__preview-icon">
+                <Field
+                  name="subject.type"
+
+                  component={field => (<Node label={field.input.value} />)}
+                />
+              </div>
+              <Field
+                name="text"
+                component={field => (
+                  <Markdown
+                    className="stage-editor-section-sociogram-prompt__preview-text"
+                    source={field.input.value}
+                  />
+                )}
+              />
+            </div>
+          </FormSection>
+        )}
+        {...rest}
+      >
+        <FormSection name={fieldId}>
+          <div className="stage-editor-section-prompt__group">
+            <Field
+              name="text"
+              component={Fields.TextArea}
+              className="stage-editor-section-prompt__setting"
+              label="Text for prompt"
+              placeholder="Enter text for the prompt here"
+            />
+          </div>
+          <div className="stage-editor-section-prompt__group">
+            <h4 className="stage-editor-section-prompt__group-title">Nodes</h4>
+            <Field
+              name="subject"
+              component={Fields.Contexts}
+              className="stage-editor-section-prompt__setting"
+              parse={value => ({ type: value, entity: 'node' })}
+              format={value => get(value, 'type')}
+              options={nodeTypes}
+              label="Which node would you like to layout?"
+            />
+            <Field
+              name="sortOrderBy"
+              component={Fields.OrderBy}
+              className="stage-editor-section-prompt__setting"
+              variables={['*', ...variablesForNodeType]}
```

Comment:
It's different in that it can be ascending/descending. Definitely confusing.

I will add some explanation to the guidance for now.

New issue: https://github.com/codaco/Architect/issues/113